salt/utils/logrotate/conf.jinja

```diff
@@ -0,0 +1,16 @@
+{%- for name, settings in logrotate_files.items() -%}
```

Comment:
I'm assuming this is intended to be `settings` instead of `logrotate_files`

Reply:
No, believe it is correct. A sample pillar file looks like the following:
```logrotate:
  test0:
    conf_file: /etc/logrotate.d/test0
    name: /var/log/test0.log
    options:
      - rotate 2
      - size 5k
      - missingok
     - postrotate systemctl restart nginx
```